lib/generators/shopify_app/add_webhook/add_webhook_generator.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails/generators/base'
+
+module ShopifyApp
+  module Generators
+    class AddWebhookGenerator < Rails::Generators::Base
+      source_root File.expand_path('../templates', __FILE__)
+
+      class_option :topic, type: :string, aliases: "-t", required: true
+      class_option :address, type: :string, aliases: "-a", required: true
+
+      def init_webhook_config
+        initializer = load_initializer
+        return if initializer.include?("config.webhooks = [")
```

Comment:
I wonder if there is better way to check if we have already configured webhooks. Hardcoding `"config.webhooks = ["` feels really fragile to me. This could break if the config code doesn't following standard ruby guidelines: `config.webhook=[`, `config.webhook= [` or `config.webhook = Array.wrap()`.